pkg/core/interop/contract/call.go

```diff
@@ -15,6 +15,26 @@ import (
 	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
 )
 
+// LoadToken calls method specified by token id.
+func LoadToken(ic *interop.Context) func(id int32) error {
+	return func(id int32) error {
+		ctx := ic.VM.Context()
+		tok := ctx.NEF.Tokens[id]
```

Comment:
NEF can easily be nil.

Reply:
It is expected, `LoadToken` should fail in this case.

Reply:
As long as the only caller is in VM's `execute` (that can recover) it's OK.

Reply:
Exactly, this is expected to be used like interop, protected by `execute`.